yearn/deploy/00_deploy_cvp_strategy.ts

```diff
@@ -0,0 +1,49 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts, helpers } = hre
+  const { deploy, log } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  const TBTCCurveVault = await deployments.getOrNull("TBTCCurveVault")
+  if (!TBTCCurveVault || !helpers.address.isValid(TBTCCurveVault.address)) {
+    throw new Error("Address of TBTCCurveVault must be set")
+  }
+
+  const TBTCCurvePoolDepositor = await deployments.getOrNull("TBTCCurvePoolDepositor")
+  if (!TBTCCurvePoolDepositor || !helpers.address.isValid(TBTCCurvePoolDepositor.address)) {
+    throw new Error("Address of TBTCCurvePoolDepositor must be set")
+  }
+
+  const TBTCCurvePoolGauge = await deployments.getOrNull("TBTCCurvePoolGauge")
+  if (!TBTCCurvePoolGauge || !helpers.address.isValid(TBTCCurvePoolGauge.address)) {
+    throw new Error("Address of TBTCCurvePoolGauge must be set")
+  }
+
+  const TBTCCurvePoolGaugeReward = await deployments.getOrNull("TBTCCurvePoolGaugeReward")
+  if (!TBTCCurvePoolGaugeReward || !helpers.address.isValid(TBTCCurvePoolGaugeReward.address)) {
+    TBTCCurvePoolGaugeReward.address = "0x0000000000000000000000000000000000000000"
```

Comment:
Why set it to zero instead of throwing an error?

Reply:
Because it's an optional value and zero address is a valid value. We don't know at this point if tBTC v2 Curve pool gauge will distribute KEEP rewards as the v1 Curve pool gauge does. We are prepared for both scenarios :)